### ISSUE TITLE
Link text on /earn isn't easily distinguishable from surrounding text.

### ISSUE THREAD

**Issue by youbrokesomething** [Automattic/wp-calypso#75604]
### Quick summary

A link on /earn is in the middle of a paragraph. When text links are surrounded by other text, they either need to have strong color contrast or styling like an underline to make them stand out from other text.
The link is:
`<a href="https://wordpress.com/support/video-tutorials-add-payments-features-to-your-site-with-our-guides/" target="_blank" rel="noopener noreferrer">Watch our tutorial videos to get started</a>`
The link text has a color contrast of 1.14:1 against the surrounding text. It is likely easier to add an underline as a one-off CSS rule.

[You can read more about link text accessibility rules for low vision in this guide.](https://dequeuniversity.com/rules/axe/4.6/link-in-text-block?application=AxeChrome)

### Steps to reproduce

1. On /earn
2. Inspect the link in the summary or use an accessibility checker tool like the axe DevTools Chrome extension.

### What you expected to happen

All links surrounded by text should be easily distinguishable from other text.

### What actually happened

There's at least one text link surrounded by other text that doesn't have enough color contrast or style to make it easy to see in the surrounding text.

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable

**Comment by allilevine** [comment]
Closed in https://github.com/Automattic/wp-calypso/pull/76896
